journal/modernized/33-testimonies.adoc

```diff
@@ -82,9 +82,7 @@ and we pray, that it may please the Almighty,
 to raise up others to supply his place and service in the churches of Christ.
 
 Signed by order, in behalf of the said meeting, by Robert Hoope, Alexander Seaton.
-
-[.asterism]
-'''
+//Does this need any signed signature stuff?
```

Comment:
Yep. Even though these are not embedded letters or testimonies, we can still use the "signed signature stuff" :) I would do it like this for all of these:
```ASCIIDOC

[.signed-section-closing]
Signed by order, in behalf of the said meeting, by 

[.signed-section-signature]
Robert Hoope, Alexander Seaton.
```

So... you'll have to scan through this chapter and apply these tags to the various testminies. 👍 And I noticed that some of them ALSO have a `[.signed-section-context-close]` after the signature.